src/ngraph_builder.cc

```diff
@@ -1516,6 +1580,7 @@ static Status TranslateDepthToSpaceOp(
   std::iota(ng_axis_order.begin(), ng_axis_order.end(), 0);
   auto reshaped =
       make_shared<ng::op::Reshape>(ng_input, ng_axis_order, ng_reshape_shape);
+  SaveNgOp(first_ng_op_map, op->name(), reshaped);
 
   auto transposed =
```

Comment:
Question: `ng::builder::numpy_transpose` seems also doing something and at the end create a ngraph reshape op, in this case, should we also consider it as an input? 
https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/builder/numpy_transpose.cpp#L77

Reply:
no, the reshape by numpy_transpose is part of the translate function so not an input.

Reply:
I see. Thanks for the confirmation!

Reply:
If the reshape is the entry node, then must be saved